src/components/ContentManager/ContentManagerContext.tsx

```diff
@@ -0,0 +1,121 @@
+import { createContext, useContext, useEffect, useMemo, useState } from 'react'
+import type { ReactNode, JSX } from 'react'
+import { useRouter } from 'tuono'
+
+/**
+ * Every page in the documentation/tutorial must be
+ * structured according to this interface.
+ */
+export interface Page {
+  /**
+   * Used for:
+   * - Sidebar navigation
+   * - Page title
+   * - Breadcrumb
+   * - Page shortcut title in root pages
+   */
+  title: string
+  /**
+   * Used for:
+   * - Generate <title> tag
+   */
+  seoTitle: string
+  /**
+   * Used for:
+   * - Generate <meta name="description"> tag
+   * - Page shortcut description in root pages
+   */
+  description: string
+  /**
+   * Used for:
+   * - Generate canonical URL
+   * - Identify current route
+   */
+  path: string
+  /**
+   * Used for:
+   * - Page content
+   */
+  pages?: Array<Page>
+}
+
+type NavigationTree = Array<Page>
+
+interface ContentManagerContextValue extends PageLocation {
+  navigationTree: NavigationTree
+}
+
+interface PageLocation {
+  currentPage: Page | undefined
+  nextPage: Page | undefined
+  previousPage: Page | undefined
+}
+
+const ContentManagerContext = createContext<ContentManagerContextValue>({
+  navigationTree: [],
+  currentPage: undefined,
+  nextPage: undefined,
+  previousPage: undefined,
+})
+
+const getPagesLocation = (
```

Comment:
The function is named using the plural form, 
but it should be singular since the result is based upon a single page.

https://github.com/tuono-labs/tuono-documentation/pull/68#discussion_r2009152046


```suggestion
const getPageLocation = (
```

Reply:
I'm not sure about that. What is returned are 3 routes `current` `next` and `previous`

Reply:
Yes, but they are all relative to the current route matched by a given path name, which corresponds to a single one.

Reply:
I don't have a strong opinion here honestly. I'll update it with your suggestion